calendar/screen.css

```
@@ -75,7 +71,35 @@ input:checked + label{
   height: 100%;
 }
 
-#calendar{
+#calendar {
+  height: 100%;
   flex-grow: 1;
   overflow: auto;
 }
+
+#calendar-options .btn.active {
+  background-color: var(--grist-theme-button-group-selected-bg);
+  color: var(--grist-theme-right-panel-toggle-button-enabled-fg);
+}
+
+/*Overriding some Toast UI styles that cannot be configured via theme*/
+.toastui-calendar-panel.toastui-calendar-time {
+  overflow-y: inherit;
+}
+
+.toastui-calendar-timegrid-time {
+  color: var(--grist-theme-text);
+}
+
+.toastui-calendar-template-alldayTitle {
+  color: var(--grist-theme-text);
+}
+
+.toastui-calendar-template-time,
+.toastui-calendar-template-monthDayName{
+  color: var(--grist-theme-text);
+}
+
+.toastui-calendar-week-view-day-names {
+  border-bottom: 1px solid var(--grist-table-body-border);
+}
```

```suggestion
  border-bottom: 1px solid var(--grist-theme-table-body-border);
}

.toastui-calendar-weekday-grid-date-decorator {
  color: var(--grist-theme-add-new-circle-small-fg) !important;
  background-color: var(--grist-theme-add-new-circle-small-bg) !important;
}

.toastui-calendar-day-view-day-names, .toastui-calendar-week-view-day-names {
  border-bottom: 1px solid var(--grist-theme-table-body-border);
}

.toastui-calendar-daygrid-cell+.toastui-calendar-daygrid-cell {
  border-left: 1px solid var(--grist-theme-table-body-border);
}

```

This should address some of the suggestions Anais made.